public/scripts/controllers/dashboardController.js

```diff
@@ -27,5 +27,17 @@ devflowApp.controller('dashboardController', ['$scope', 'userService', 'requestS
 
         $scope.links = config.links;
 
+        $scope.showSection = function(sectionName) {
+            switch(sectionName) {
+                case 'pullrequests':
+                    return $scope.pullrequests && $scope.pullrequests.length > 0 && $scope.isFeatureEnabled(sectionName);
```

Comment:
Some of these conditions could benefit from parentheses, per our style. Ex: `($scope.pullrequests.length > 0)`